server/views/partials/layout.njk

```diff
@@ -70,6 +70,23 @@
 {% endblock %}
 
 {% block bodyEnd %}
+  {% if sentry and sentry.loaderScriptId %}
+    <script nonce="{{ cspNonce }}">
+      window.sentryOnLoad = function() {
+        Sentry.init({
+          release: "hmpps-manage-a-supervision-ui@{{ version }}",
```

Comment:
Does this name need to be changed to something like hmpps-challenge-support-intervention-plan-ui ?